atd-vze/src/views/Crashes/Crash.js

```diff
@@ -220,9 +204,7 @@ function Crash(props) {
         </Col>
         <Col xs="12" sm="6" md="4">
           <Widget02
-            header={`${
-              totalYearsOfLifeLost === null ? "--" : totalYearsOfLifeLost
-            }`}
+            header={`${yearsOfLifeLost === null ? "--" : yearsOfLifeLost}`}
```

Comment:
interesting to see `--`. i think you can simplify this to `yearsOfLifeLost || 0`. i don't know why we would prefer `--` over `0`

Reply:
done! i dont even think its possible for yearsOfLifeLost to be null with the way its calculated but its good to have this catch just in case